ftw/mobilenavigation/browser/navigation.py

```diff
@@ -33,10 +34,11 @@ def __call__(self):
             # makes the request be fired on the default page.
             context = aq_parent(aq_inner(context))
 
-        subnavi = '<ul>'
+        subnavi = u'<ul>'
         level = int(self.request.form.get('level', '1'))
         if level == 0:
-            subnavi = '<ul id="portal-globalnav" class="mobileNavigation">'
+            subnavi = u'<ul id="portal-globalnav" class="mobileNavigation">'
+        import pdb
```

Comment:
💥 - without the `pdb.set_trace()` I think there's a discount - its only half a füflliber 😜